daemons/controld/controld_control.c

```diff
@@ -326,29 +326,24 @@ do_exit(long long action,
     crmd_exit(exit_code);
 }
 
-static void sigpipe_ignore(int nsig) { return; }
-
-/*	 A_STARTUP	*/
+// A_STARTUP
 void
-do_startup(long long action,
-           enum crmd_fsa_cause cause,
-           enum crmd_fsa_state cur_state, enum crmd_fsa_input current_input, fsa_data_t * msg_data)
+do_startup(long long action, enum crmd_fsa_cause cause,
+           enum crmd_fsa_state cur_state, enum crmd_fsa_input current_input,
+           fsa_data_t *msg_data)
 {
-    crm_debug("Registering Signal Handlers");
     mainloop_add_signal(SIGTERM, crm_shutdown);
-    mainloop_add_signal(SIGPIPE, sigpipe_ignore);
+    mainloop_add_signal(SIGPIPE, NULL); // Ignore SIGPIPE
```

Comment:
Please also modify the doxygen on `mainloop_add_signal` to reference that the `dispatch` param can be NULL.